osler/templates/workup/basicnote_detail.html

```diff
@@ -0,0 +1,35 @@
+{% extends "core/base.html" %}
+
+{% block title %}
+Basic Note: {{ basicnote.patient.name }}
+{% endblock %}
+
+{% block header %}
+  <div class="row">
+    <div class="col-md-10">
+      <h2>{{ basicnote.title }}</h2>
+      <p class="lead">For <a href="{% url 'core:patient-detail' pk=basicnote.patient.id %}">{{ basicnote.patient.name }}</a></p>
+    </div>
+    {% if can_modify_basicnote %}
+    <div class="col-md-1 pull-right">
+    <h1><a href="{% url 'attestable-basic-note-update' pk=basicnote.id %}">
```

Comment:
make sure this is just basic-note-update

Reply:
thanks for noticing!